pkg/query-service/app/dashboards/model.go

```diff
@@ -532,11 +531,35 @@ func extractDashboardName(data map[string]interface{}) string {
 	return ""
 }
 
-func countPanelsInDashboard(data map[string]interface{}) model.DashboardsInfo {
-	var logsPanelCount, tracesPanelCount, metricsPanelCount int
+func checkLogPanelAttrContains(data map[string]interface{}) int {
+	var logsPanelsWithAttrContains int
+	filters, ok := data["filters"].(map[string]interface{})
+	if ok && filters["items"] != nil {
+		items, ok := filters["items"].([]interface{})
+		if ok {
+			for _, item := range items {
+				itemMap, ok := item.(map[string]interface{})
+				if ok {
+					if slices.Contains([]string{"contains", "ncontains", "like", "nlike"}, itemMap["op"].(string)) {
```

Comment:
add assertion for `itemMap["op"].(string)` otherwise it will panic.

Reply:
thanks 👍